### ISSUE TITLE
Use with gulp-sourcemaps?

### ISSUE THREAD

**Issue by evillemez** [mikach/gulp-concat-sourcemap#11]
I just want to clarify that this is in fact the plugin the community should be using in conjunction with `gulp-sourcemaps`.  From scanning the code, this seems to be the case - but I'm not entirely sure.  The state of sourcemaps is a little hard to follow at the moment.

Is this generally what I should be doing?

``` js
var gulp = require('gulp')
  , coffee = require('gulp-coffee')
  , concat = require('gulp-concat-sourcemap')
  , gutil = require('gulp-util')
;

gulp.task('build', function() {
  gulp.src('**/*.coffee')
      .pipe(sourcemaps.init())
        .pipe(coffee({bare: true})).on('error', gutil.log)
        .pipe(concat('bundle.js', {sourcesContent: true})).on('error', gutil.log)
      .pipe(sourcemaps.write())
      .pipe(gulp.dest(target));
});
```

**Comment by ghost** [comment]
any clue on this? I'm not sure how to make sourcemaps work properly with coffeescript!
